asserts/database.go

```diff
@@ -465,13 +463,11 @@ func (db *Database) Check(assert Assertion) error {
 		}
 	}
 
-	var delegationConstraints []*AssertionConstraints
 	for _, checker := range db.checkers {
-		acs, err := checker(assert, accKey, delegationConstraints, db, earliestTime, latestTime)
+		err := checker(assert, accKey, db, earliestTime, latestTime)
```

Comment:
(nitpick^9) we could write this as `if err := ..; er != nil` but what you done here was how things looked before the auth delegation PRs landed so I think it's the right call.